spec/stormpath/client_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe Stormpath::Rails::Client, vcr: true do
+  let(:default_base_url) { 'https://api.stormpath.com/v1' }
+  let(:custom_base_url) { 'https://eu.stormpath.io/v1' }
+  let(:stormpath_client) { Stormpath::Rails::Client }
+  let(:api_key) { Stormpath::Rails::ApiKey.new }
+
+  describe 'custom base url set' do
```

Comment:
`custom base url set without permissions` - because it seems this way that any way you would get an error. After this just merge it in and release a new version